### PR TITLE
 http://darktable.sf.net -> https://www.darktable.org/ 

### PR DESCRIPTION
**Pull request by supertobi**

### CODE REVIEW

File: src/external/CL/permission [hunk 1]
```diff
@@ -63,7 +63,7 @@ header files with an open source project called "Darktable".
 Darktable is a photography workflow solution for Linux with a virtual=20
 lighttable and darkroom. Its main focus and strength is in conversion and=20
 development of photographic RAW-files. Details can be found at=20
-http://darktable.sourceforge.net. Darktable is distributed in source code a=
```
Comment (LebedevRI) [inline src/external/CL/permission]: Pretty sure you should not be changing this.

File: packaging/opensuse/darktable.spec [hunk 2]
```diff
@@ -19,7 +19,7 @@
 Name:           darktable
 Version:        1.0.3
```
Comment (LebedevRI) [inline packaging/opensuse/darktable.spec]: Why is this file even here?
It's completely out-of-date.

File: src/external/CL/permission (hunk 1 above)
Comment (supertobi) [reply src/external/CL/permission]: You are right.

**Comment by supertobi** [comment]
packaging/opensuse/darktable.spec will be deleted, so I close this PR

**Pull request closed without merging**